state/account.go

```diff
@@ -147,6 +147,12 @@ func (st *Account) AccountType() int32 {
 
 // SetPendingNonce sets the pending nonce
 func (st *Account) SetPendingNonce(nonce uint64) error {
+	// this is a legacy account that had never initiated an outgoing transaction
+	// so we can convert it to zero-nonce account
+	if st.accountType == 0 && st.nonce == 0 && nonce == 1 {
+		st.accountType = 1
+	}
```

Comment:
should be a hard-fork?

Reply:
https://github.com/iotexproject/iotex-core/pull/4163/files#diff-8d0192f25f9daf6c1bfa70c3b75949444fe5d1d47193e6b7c1210c39a19ed52aR392